README.md

```diff
@@ -596,6 +596,36 @@ const posts = await postRepository.find({
 });
 ```
 
+### Operator `match`
```

Comment:
Can we add a link to this header at line 552?

> [`match`](#operator-match)

Reply:
done